__tests__/e2e.test.ts

```diff
@@ -79,8 +87,8 @@ beforeAll(
       { concurrency: 1 },
     );
   },
-  // 5 minute timeout for pnpm installation inside test repos
-  300000,
+  // 7.5 minute timeout for pnpm installation inside test repos
+  450000,
```

Comment:
we have to increase the timeout because it takes longer for pnpm to install on Windows in `tmpdir()` apparently